assets/src/application/ui/input/Button/Link.tsx

```diff
@@ -5,20 +5,20 @@ import { Tooltip } from '@infraUI/display';
 import { LinkProps } from './types';
 import './style.scss';
 
-const Link: React.FC<LinkProps> = ({ external, href, icon, tooltip, tooltipProps, ...props }) => {
+const Link: React.FC<LinkProps> = ({ children, external, href, icon, tooltip, tooltipProps, ...props }) => {
 	const className = classNames('ee-btn-base ee-icon-button ee-icon-button--borderless', props.className);
 
-	if (external && icon) {
+	if (external && tooltip) {
```

Comment:
not grokking what `external` is intending to do the way things are set up now...
currently it appears to do nothing but control whether the link renders or not.

I'm wondering if it was maybe originally intended to control the display of the `Link` icon
and would be used to indicate that a link will take the user to another website.

As well, if the link does not have a tooltip then nothing is rendered, which also doesn't seem right.

Maybe this component should be something like this instead:

```js
const Link: React.FC<LinkProps> = ({ children, external, href, icon, tooltip, tooltipProps, ...props }) => {
	const className = classNames('ee-btn-base ee-icon-button ee-icon-button--borderless', props.className);
	const linkOnly = (
		<a href={href} className={className} target='_blank' rel='noopener norefferer'>
			{icon ? icon : children}
			{external ? <LinkIcon /> : null}
		</a>
	);
	if (tooltip) {
		return (
			<Tooltip tooltip={tooltip} {...tooltipProps}>
				{linkOnly}
			</Tooltip>
		);
	}
	return { linkOnly };
};
```

but ya know... done correctly (since that was whipped together quickly)

Reply:
originally, I thought that we might abstract the `Link` from `react-router` also in this component, but later found out that we won't use that library, and that's why I left that comment that it might be extended later, because I wasn't sure what other cases it should also handle

Reply:
since that anchor element has `target='_blank' rel='noopener norefferer'` that means that it always will be external, so I've removed external prop;
your suggestion looks great, updated with some changes: https://github.com/eventespresso/event-espresso-core/pull/3022/commits/f45bedc09117fec7979a36cb30e445b9fee952bd

Reply:
I did not added this
`{external ? <LinkIcon /> : null}`
because that icon needs to be customized, as in `RegistrationsLink` with `Groups` icon

Reply:
> since that anchor element has target='_blank' rel='noopener norefferer' that means that it always will be external

well `target='_blank'` just means the link will open in another tab or window, but my understanding of that "external" icon that appears after some links ios that it is supposed to indicate that the link takes you to another site altogether


> I did not added this `{external ? <LinkIcon /> : null}`

ya no worries, I didn't expect those suggested changes to be the final code and fully expected additional changes and/or fixes

Reply:
> "external" icon that appears after some links ios that it is supposed to indicate that the link takes you to another site altogether
yes, you are right, but currently we are not covering cases when we open a link of another site, nor do we have that external icon I posted yesterday, that's why I remove `external` prop for the moment to avoid any confusion and inconsistencies